airdrops/index_v1.json

```diff
@@ -191,6 +191,23 @@
                 "chain_id": 8453,
                 "decimals": 18
             }
+        },
+        "omni": {
+            "csv_path": "airdrops/omni.csv",
+            "csv_hash": "47e07a2da379ace4aff475ab9e2e60fa870aaaaac981715a807ede30a6e1c8c3",
+            "asset_identifier": "eip155:1/erc20:0x36E66fbBce51e4cD5bd3C62B637Eb411b18949D4",
+            "url": "https://claims.omni.network/",
+            "name": "OMNI",
+            "icon": "omni.svg",
+            "icon_path": "airdrops/icons/omni.svg",
+            "new_asset_data": {
+                "asset_type": "EVM_TOKEN",
+                "address": "0x36e66fbbce51e4cd5bd3c62b637eb411b18949d4",
```

Comment:
Sorry I just noticed that the address is not checksummed here. It should be `0x36E66fbBce51e4cD5bd3C62B637Eb411b18949D4`. We should have a tests also for this